example/dashbling.config.js

```diff
@@ -20,6 +20,22 @@ module.exports = {
     // start custom code that sends events here,
     // for example listen to streams etc.
   },
+  configureServer: hapiServer => {
+    // Configure the Hapi server here.
+    // See https://hapijs.com/api/17.1.1 docs for details.
+    // This is only needed for more advanced use cases.
+    hapiServer.route({
+      method: "GET",
+      path: "/ping",
+      handler: (_request, _h) => {
+        return "pong";
+      }
+    });
+
+    // Be sure to return a Promise, so the initialization process
+    // waits for this configuration to be completed.
+    Promise.resolve();
```

Comment:
@pascalw I know this is just an example, but should it be `return Promise.resolve()`?

Reply:
Good catch! I've been doing to much ReasonML lately haha 🙂

Reply:
I've updated the example to use an async function, so you don't have to explicitly return a promise.

Reply:
👍